src/main/java/com/xebialabs/deployit/ci/Credential.java

```diff
@@ -241,6 +242,13 @@ public static StandardUsernamePasswordCredentials lookupSystemCredentials(String
         );
     }
 
+    public static StandardUsernamePasswordCredentials lookupSystemCredentials(String credentialsId, Item itemGroup) {
+        return CredentialsMatchers.firstOrNull(
+                lookupCredentials(StandardUsernamePasswordCredentials.class, itemGroup, ACL.SYSTEM,
```

Comment:
This code is almost the same as the other lookupSystemCredentials , can we make the call at one place and delegate to that from another. I think ItemGroup and Jenkins should be in the same hierarchy.